cdap-api/src/main/java/co/cask/cdap/api/workflow/AbstractWorkflowAction.java

```diff
@@ -49,8 +50,8 @@ public void destroy() {
     // No-op
   }
 
-  protected final WorkflowContext getContext() {
-    return context;
+  public WorkflowContext getContext() {
```

Comment:
we need this public because the user needs to call getContext() in order to use getServiceURL for example